src/main.nim

```diff
@@ -5858,9 +5876,17 @@ proc handleGlobalKeyEvents(a) =
 
 
       elif ke.isShortcutDown(scJumpToLinkedCell):
-        let otherLoc = map.getLinkedLocation(cur)
-        if otherLoc.isSome:
-          moveCursorTo(otherLoc.get, a)
+        let otherLocs = map.getLinkedLocations(cur)
+        if otherLocs.len == 1:
+          moveCursorTo(otherLocs.first.get, a)
+        elif otherLocs.len > 1:
+          ui.jumpToSrcLocation = cur
+          ui.jumpToDestinations = otherLocs.toSeq
```

Comment:
Yeah, so with src and dest swapped in the names this code would make a lot more sense (we support single destinations and multiple sources).